### DISCUSSION TITLE
Using Fooocus inpainting model in Forge?

### DISCUSSION THREAD

**Discussion by Melyns** [lllyasviel/stable-diffusion-webui-forge#1552] [General]
I'm trying to use Fooocus inpaint model, is it achievable? i read that it's possible to do in comfy, i just don't know what i'm missing here because the inpaint results are either noisy or black. I tried almost all inpaint setting combinations/modes. 
![image](https://github.com/user-attachments/assets/dd21798c-1097-4394-bd64-546932d208a7)

**Comment by kalle07** [comment]
i see your new idea ...
iam only a user ... but why you dont use inpaint models (SDXL and PONY), they are not that different if you have each of them...
and if iam right fooocus use this model (v26) to convert the checkpoint-model you have choosen ... (so thats not an inpaint-model)

**Comment by mold0101** [comment]
With inpaint_v26.fooocus use inpaint_global_harmonius. Clean the prompt of any lora or leave it blank (and of course "Resize and Fill" and "Controlnet is more important")

EDIT: apparently it only works the first time and then it gives only a garbled image or a black screen. Restarting the UI give every time another one shot.

EDIT 2: switching to another model it's a faster way to reset the clock probably because the memory manager cleans up the leak.

**Comment by rlr0022** [comment]
I'm having the same issue. I have reverted to an earlier version from a few weeks ago until this is fixed.

**Comment by zombiehellmonkey** [comment]
You have to use a Timestep Range that is less than 0.7, in Fooocus it defaults to 0.5, so 0.5 is a good starting point. Use inpaint_global_harmonious, inpaint_only, inpaint_only+lama.
[Reference link](https://github.com/lllyasviel/stable-diffusion-webui-forge/commit/fb2e2716683453e8b0ba775f7596d8c9b080177d)

**Reply by mold0101** [reply]
If I put time step range 0,5, at 0.5 it start applying the denoise to the whole image, which is not the wanted result.

**Reply by zombiehellmonkey** [reply]
It shouldn't do at all, especially if you have a mask.

**Comment by technicalstorm** [comment]
It would be awesome, to be able to use Fooocus inpainting in Forge

**Reply by ride5k** [reply]
does this work or no?
